examples/specs/layer_bar_annotations.vl.json

```diff
@@ -0,0 +1,60 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+  "description": "A simple bar chart with annotation.",
+  "data": {
+    "values": [
+      {"Day": "D01", "Value": "50"},
+      {"Day": "D02", "Value": "110"},
+      {"Day": "D03", "Value": "70"},
+      {"Day": "D04", "Value": "50"},
+      {"Day": "D05", "Value": "80"},
+      {"Day": "D06", "Value": "150"},
+      {"Day": "D07", "Value": "140"},
+      {"Day": "D08", "Value": "130"},
+      {"Day": "D09", "Value": "390"},
+      {"Day": "D10", "Value": "200"},
+      {"Day": "D11", "Value": "70"},
+      {"Day": "D12", "Value": "50"},
+      {"Day": "D13", "Value": "100"},
+      {"Day": "D14", "Value": "150"},
+      {"Day": "D15", "Value": "130"},
+      {"Threshold": "hazardous", "ThresholdValue": "300"}
```

Comment:
I don't think you should make the data table heterogenous.  In a way, you should think about this as one data source for the bar and perhaps another with only one data point for the threshold.
In a way, we might not really need the second data source, as it's more like constant value.  

In any case, please remove this line from the main data source?